### PR TITLE
feat: [CET-1225]: CET active agents card in subscription page

### PR DESCRIPTION
**Pull request by guycastel**
#### Screenshots

(Forgot to take screenshot before PR env deleted 🤷)

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by guycastel** [comment]
retrigger titlecheck

**Comment by guycastel** [comment]
retrigger jest

**Review by ricksjames — APPROVED** [review]

**Review by tan-nhu — DISMISSED** [review]

File: src/modules/10-common/strings/strings.en.yaml [hunk 1]
```diff
@@ -539,6 +539,8 @@ subscriptions:
     developers: '{{ developers }} Developers'
   cet:
     agents: '{{ agents }} Agents'
+    whatIsActiveAgent: What is an Active Agent?
+    whatIsActiveAgentTooltip: The agent actively monitors the Java applications for run-time exceptions, log events, and any custom events that are configured by the user.
```
Comment (cseverino789) [inline src/modules/10-common/strings/strings.en.yaml:543]: We already have similar Strings for the Active Agent tool tip...

https://github.com/harness/harness-core-ui/blob/ce5e67c88c1fd56697ee3eeaa71ffb1752c40717/src/modules/10-common/strings/strings.en.yaml#LL587C35-L587C35

  cetAgentToolTip: An agent is an independent unit of software that you install on a host or in a container that tracks for errors continuously and sends the error tracking data to the Harness Platform.**

**Review by cseverino789 — CHANGES_REQUESTED** [review]

**Comment by guycastel** [comment]
retrigger cypress-cv

**Review by dhruvUpadhyay07 — APPROVED** [review]

**Pull request merged**